### ISSUE TITLE
Workaround #1001 for Fennec breaks Google Hangout extension

### ISSUE THREAD

**Issue by gorhill** [uBlock-LLC/uBlock#1025]
As reported in Chrome store feedback:

> when uBlock is active in current window Hangouts Extension loads to an error message saying it's blocked by an Extension. Disabling uBlock in the current window allows it to reload, enabling it again block it again. Silly behavior... obviously uBlock shouldn't be blocking access to other third party Chrome Extensions, just the content on the page.

Fixing.

**Comment by gorhill** [comment]
Fixed in 1d3dd12fab5323627747ddf8100278b380425dd7.

**Comment by gorhill** [comment]
Now that I have a bit more time... The problem was because for whatever reason Google Hangout is created without a `main_frame`. Thus as per #1001, uBlock would assign the URL of the last seen `main_frame` as the main frame of Google Hangout, and for those who happen to block 3rd-party frames, this would cause Google Hangout subframe to be blocked.

So the workaround to the workaround is to limit the lifetime of the last seen `main_frame`. I just set half a second for now, it was an emergency fix so I had to come up with something quickly.
